Control/public/app.css

```diff
@@ -116,3 +116,4 @@
 .grid-container { display: grid; gap: 1  rem; grid-template-columns: repeat(auto-fill, minmax(150px, 1fr)); }
 .bg-yellow { background-color: #e6e200; }
 .italic { font-style: italic; }
+.left-align { display: flex; margin-left: auto; }
```

Comment:
I think that `display: flex` should not be part of a class named `left-align`, and I am pretty sure you do not need it there

Reply:
Indeed, thank you!